src/main/java/lotto/domain/LottoTicketFactory.java

```diff
@@ -0,0 +1,41 @@
+package lotto.domain;
+
+import lotto.domain.numbers.LottoNumber;
+import lotto.domain.numbers.LottoTicket;
+
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class LottoTicketFactory {
+    private final static int LOTTO_START_NUMBER = 1;
+    private final static int LOTTO_END_NUMBER = 45;
+    private final static int SPLIT_START_INDEX = 0;
+    private final static int SPLIT_END_INDEX = 6;
```

Comment:
```suggestion
    private static final int LOTTO_START_NUMBER = 1;
    private static final int LOTTO_END_NUMBER = 45;
    private static final int SPLIT_START_INDEX = 0;
    private static final int SPLIT_END_INDEX = 6;
```

다른 코드에도 순서를 잘못 쓴 상수가 있는 것 같은데 같이 확인해주세요!

Reply:
네! 제가 실수했네요!!ㅠ